security/cert_manager_operator/cert-manager-operator-proxy.adoc

```diff
@@ -0,0 +1,16 @@
+:_content-type: ASSEMBLY
+[id="cert-manager-operator-proxy"]
+= Configuring the egress proxy for the {cert-manager-operator}
+include::_attributes/common-attributes.adoc[]
+:context: cert-manager-operator-proxy
+
+toc::[]
+
+TODO
+
+// TODO: Need more details about what we're doing/why this is needed
```

Comment:
@thejasn @swghosh There was some explanation in the doc you shared here, but I didn't really get what exactly we're doing here, and why this is needed? Can you give me any more details here?

Reading https://docs.openshift.com/container-platform/4.12/operators/admin/olm-configuring-proxy-support.html, it says:

> If a global proxy is configured on the OpenShift Container Platform cluster, Operator Lifecycle Manager (OLM) automatically configures Operators that it manages with the cluster-wide proxy. 

So I wasn't sure why manual intervention here is needed.

Reply:
The same docs you linked also mentions, 
> Operators must handle setting environment variables for proxy settings in the pods for any managed Operands.

Along with this, we are also covering `Injecting a custom CA certificate`, which is also covered in this section. This doc explains the same procedure (same content and steps) provided in https://docs.openshift.com/container-platform/4.12/operators/admin/olm-configuring-proxy-support.html but slightly more specific to cert-manager-operator.

Reply:
Also most of content [here](https://github.com/openshift/cert-manager-operator/blob/master/docs/proxy.md) is related to `Injecting a custom CA certificate`.